### PR TITLE
Allow service to be used only for tracing

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <p.loffay@gmail.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -230,9 +230,9 @@ func main() {
 		stdlog.Fatalf("parse flag: %v", err)
 	}
 
-	if !cfg.metrics.enabled && !cfg.logs.enabled {
+	if !cfg.metrics.enabled && !cfg.logs.enabled && !cfg.traces.enabled {
 		// Currently we don't support the case where only the gRPC trace endpoint is enabled
```
Comment (bwplotka) [inline main.go:234]: This comment is not obsolete, right?

**Review by bwplotka — APPROVED** [review]
Thanks 👍🏽 One nit.

File: main.go (hunk 1 above)
Comment (pavolloffay) [reply main.go:234]: I don't know, it's not the subject of this PR.

Comment (pavolloffay) [reply main.go:234]: actually yes, let me remove it

**Review by bwplotka — APPROVED** [review]

**Pull request merged**